src/SFML/Audio/SoundBuffer.cpp

```diff
@@ -52,10 +52,7 @@ SoundBuffer::SoundBuffer()
 
 
 ////////////////////////////////////////////////////////////
-SoundBuffer::SoundBuffer(const SoundBuffer& copy) :
-m_samples(copy.m_samples),
-m_duration(copy.m_duration),
-m_sounds() // don't copy the attached sounds
```

Comment:
I'd like to preserve this comment after the `m_duration` initializer since it's otherwise not obvious why the code is written this way.